File: huannv_Rx_Retrofit_MultiThread/app/src/main/java/com/example/base_project/detailFim/model/AllDetailFilmResponse.java
package com.example.base_project.detailFim.model;

import com.example.base_project.upcoming.model.MovieResponse;

public class AllDetailFilmResponse {
    private GetVideoResponse getVideoResponse;
    private GetDetailResponse getDetailResponse;
    private MovieResponse getSimilarResponse;
    private MovieResponse getRecommendation;

    public MovieResponse getGetSimilarResponse() {
        return getSimilarResponse;
    }

    public void setGetSimilarResponse(MovieResponse getSimilarResponse) {
        this.getSimilarResponse = getSimilarResponse;
    }

    public MovieResponse getGetRecommendation() {
        return getRecommendation;
    }

    public void setGetRecommendation(MovieResponse getRecommendation) {
        this.getRecommendation = getRecommendation;
    }

    public GetDetailResponse getGetDetailResponse() {
        return getDetailResponse;
    }

    public void setGetDetailResponse(GetDetailResponse getDetailResponse) {
        this.getDetailResponse = getDetailResponse;
    }

    public GetVideoResponse getGetVideoResponse() {
        return getVideoResponse;
    }

    public void setGetVideoResponse(GetVideoResponse getVideoResponse) {
        this.getVideoResponse = getVideoResponse;
    }
}

File: Trainning_Kotlin/app/src/main/java/com/example/myapplication/detailFilm/model/AllDetailFilmResponse.kt
package com.example.myapplication.detailFilm.model

import com.example.myapplication.model.MovieResponse

class AllDetailFilmResponse {
    private var getVideoResponse: GetVideoResponse? = null
    private var getDetailResponse: GetDetailResponse? = null
    private var getSimilarResponse: MovieResponse? = null
    private var getRecommendation: MovieResponse? = null

    fun getGetSimilarResponse(): MovieResponse? {
        return getSimilarResponse
    }

    fun setGetSimilarResponse(getSimilarResponse: MovieResponse?) {
        this.getSimilarResponse = getSimilarResponse
    }

    fun getGetRecommendation(): MovieResponse? {
        return getRecommendation
    }

    fun setGetRecommendation(getRecommendation: MovieResponse?) {
        this.getRecommendation = getRecommendation
    }

    fun getGetDetailResponse(): GetDetailResponse? {
        return getDetailResponse
    }

    fun setGetDetailResponse(getDetailResponse: GetDetailResponse?) {
        this.getDetailResponse = getDetailResponse
    }

    fun getGetVideoResponse(): GetVideoResponse? {
        return getVideoResponse
    }

    fun setGetVideoResponse(getVideoResponse: GetVideoResponse?) {
        this.getVideoResponse = getVideoResponse
    }
}